arm/:
  arm_node:
    ros__parameters:
      config_package: "hebi_description"
      config_file: "config/A-2085-06.cfg.yaml"

omni_base_node:
  ros__parameters:
    names: [ "wheel_1", "wheel_2", "wheel_3" ]
    families: [ "Rosie" ]
    publish_odom: true